exampleSite/content/en/usage/configuration.md

```diff
@@ -269,7 +269,8 @@ params:
 # Set type to 'posts' if you want to render page as blogpost
 type = "posts"
 
-# Set page weight to re-arrange items in file-tree menu.
+# Set page weight to re-arrange items in file-tree menu. Higher weight means
```

Comment:
Are `float` weights allowed?

Reply:
This is a Hugo option, and I don't want to duplicate the documentation for it. If necessary, we can add a link to https://gohugo.io/content-management/front-matter/?search-input=weight#front-matter-variables.

Reply:
Okay, I will update to include this link tomorrow

Reply:
Done, lemme know if you have further requests here